backend/src/router.js

```diff
@@ -2,12 +2,77 @@ const express = require("express");
 
 const router = express.Router();
 
-const itemControllers = require("./controllers/itemControllers");
-
-router.get("/items", itemControllers.browse);
-router.get("/items/:id", itemControllers.read);
-router.put("/items/:id", itemControllers.edit);
-router.post("/items", itemControllers.add);
-router.delete("/items/:id", itemControllers.destroy);
+const questions = {
```

Comment:
this should be in database :)

if you want to first scaffold the form, move this in front end in your component